### ISSUE TITLE
Training from different backbone

### ISSUE THREAD

**Issue by 361720691** [JDAI-CV/DCL#20]
I want to train Stanford dog dataset with DCL from a pretrained model, when I start my training job like this:
```shell
python train.py --data product --epoch 360 --backbone resnet50 \
                    --tb 16 --tnw 16 --vb 512 --vnw 16 \
                    --lr 0.0008 --lr_step 60 \
                    --cls_lr_ratio 10 --start_epoch 1 \
                    --detail resnet50_zqs --size 512 \
                    --crop 448 --cls_mul --swap_num 7 7
```
It works just fine, I got a 84% accuracy. So I try to train it with a larger backbone like this:
```shell
python train.py --data product --epoch 60 --backbone senet154 \
                    --tb 16 --tnw 32 --vb 512 --vnw 32 \
                    --lr 0.01 --lr_step 12 \
                    --cls_lr_ratio 10 --start_epoch 1 \
                    --detail senet154_zqs --size 512 \
                    --crop 448 --cls_2 --swap_num 7 7
```
However, it ended with 29% accuracy. I don't know wether there are something wrong with my hyper parameter, can anyone help ?

I have already downloaded the pretrained model and put it in the correct path, and in config.py i added it like this:
pretrained_model = {'resnet50': './models/pretrained/resnet50-19c8e357.pth',
                    'senet154': './models/pretrained/senet154-c7b49a05.pth'}

**Comment by akira-l** [comment]
Maybe loss or accuracy curves are helpful. In this case, it's more like your senet154 is overfitted.

**Comment by 361720691** [comment]
> Maybe loss or accuracy curves are helpful. In this case, it's more like your senet154 is overfitted.

In fact, after first epoch, the accuracy of resnet50 comes to 69%, while senet154 only gets 0.58%. And after 100 epochs, the loss of 'resnet 50' comes to 0.3 (at the beginning it was 10) with 84% accuracy, the loss of 'senet154' comes to 2.8(at the beginning it was 7) with 29% accuracy. And both of their losses are steady. So I think maybe it's not overfitted, but something wrong with the hyper parameters.

**Comment by lyliuyan1995** [comment]
how to test?when i use the train.py , how to set the relevant parameters?

**Comment by fantaojie** [comment]
I have encountered a problem similar to yours. Can you share how you solved it?

**Comment by akira-l** [comment]
Senet154 may not fit the dataset or the problem.  More experiences in choosing backbones and setting hyperparameters are welcome. We'd like to receive more results from applications of DCL.
